README.md

```diff
@@ -64,12 +64,22 @@ The list of actions will likely grow as we get feedback from you.
 
 ## Install ⬇️
 
-### Running script (Recommended)
-
+### Running script (Recommended):
 ```bash
 bash <(curl -Ls https://install.tuist.io)
 ```
 
+
+### Using [Homebrew](http://brew.sh/):
+```
+brew install tuist
+```
+
+### Using [Mint](https://github.com/yonaskolb/mint)
+```
+$ mint install tuist/tuist
```

Comment:
This was attempted in the past:
https://github.com/tuist/tuist/issues/315

Sadly it won't work out of the box due to tuist requiring an additional dynamic library `ProjectDescription` to be installed alongside it and if I recall correctly Mint only installs the executables?

Reply:
yeah, you are right, I've got. So I will change pull request to "Fix warnings"